api/src/pcapi/routes/public/individual_offers/v1/events.py

```diff
@@ -445,14 +445,14 @@ def patch_event_price_category(
         )
     ),
 )
-def post_event_stocks(event_id: int, body: serialization.DatesCreation) -> serialization.PostDatesResponse:
+def post_event_stocks(event_id: int, body: serialization.EventStocksCreation) -> serialization.PostDatesResponse:
     """
     Add Stocks to an Event
 
     Add stocks to given event. Each stock is attached to a price category and to a date.
     For a given date, you will have one stock per price category.
 
-    **⚠️ Event must have less than 1 000 stocks** otherwise they will not be published.
+    **⚠️ Event must have less than 2 500 stocks** otherwise they will not be published.
```

Comment:
je sais pas si c'est très catholique mais on peut pas utiliser une f-string ici plutôt ? 
```suggestion
    **⚠️ Event must have less than {Offer.MAX_STOCKS_PER_OFFER} stocks** otherwise they will not be published.
```

Reply:
En fait ça casse la génération du JSON Open API de faire ça.